### ISSUE TITLE
Reversible flag odd results

### ISSUE THREAD

**Issue by denjots** [lucidrains/se3-transformer-pytorch#4]
Sorry if it's expected behaviour again, but with a slight tweak of your new example code I get unexpected results when your new reversible option is used to return type 1 data. Here's the code I'm running...

```
import torch
from se3_transformer_pytorch import SE3Transformer

model = SE3Transformer(
    num_tokens = 20,
    dim = 32,
    dim_head = 32,
    heads = 4,
    depth = 12,             # 12 layers
    input_degrees = 1,
    num_degrees = 2,
    output_degrees = 2,
    reduce_dim_out = True,
    reversible = True       # set reversible to True
).cuda()

atoms = torch.randint(0, 4, (1, 50)).cuda()
coors = torch.randn(1, 50, 3).cuda()
mask  = torch.ones(1, 50).bool().cuda()

pred = model(atoms, coors, mask = mask, return_type = 1)

loss = pred.mean()
print(loss)
loss.backward()

```

Without the reversible flag, the loss is close to zero as might be expected as the input coords are centered on the origin. However, when reversible is set, a very high value is produced which is going to blow up training if this was for real. Is this an expected side effect of reversible nets? Is some kind of normalization essential in that case? Or am I just doing something wrong here?

**Comment by lucidrains** [comment]
@denjots that was quick! didn't expect anyone to use this immediately

so I played around with it, and it seems like around depth 8 - 10, it starts to explode. I decided to add a final normalization, which is customary for most transformers these days https://github.com/lucidrains/se3-transformer-pytorch/releases/tag/0.0.17 should be better!

**Comment by denjots** [comment]
Ha ha! Sorry, just was checking something and noticed the update - thought I'd give it a spin. Yes, that fixes it nicely - thanks for the ultra rapid fix.

**Comment by lucidrains** [comment]
You caught me at the right time
